inference-engine/src/transformations/include/transformations/mish_fusion.hpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2018-2020 Intel Corporation
```

Comment:
Add detailed description for this transformation. You can find an example here: https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/transformations/include/transformations/optimize_strided_slice.hpp#L39